### PR TITLE
TCVP-44 ( Add code climate config)

### PR DESCRIPTION
**Pull request by rantifamilusi**
# Description

This PR includes the following proposed change(s):

- https://justice.gov.bc.ca/jira/browse/TCVP-44

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/citizen-portal/src/app/app.component.ts [hunk 1]
```diff
@@ -8,5 +8,5 @@ import { Component } from '@angular/core';
 export class AppComponent {
   title = 'Traffic Court';
 
-  constructor() {}
+  // constructor() {}
```
Comment (pbolduc) [inline src/frontend/citizen-portal/src/app/app.component.ts:11]: Why is this commented out. If it is not required, it should be removed.

File: src/frontend/citizen-portal/src/app/core/services/logger.service.ts [hunk 2]
```diff
@@ -3,10 +3,10 @@ import { Injectable } from '@angular/core';
 import { environment } from '@env/environment';
 
 @Injectable({
-  providedIn: 'root'
+  providedIn: 'root',
 })
 export class LoggerService {
-  constructor() { }
+  // constructor() { }
```
Comment (pbolduc) [inline src/frontend/citizen-portal/src/app/core/services/logger.service.ts:9]: Same as above

File: src/frontend/citizen-portal/src/app/core/services/logger.service.ts [hunk 3]
```diff
@@ -3,10 +3,10 @@ import { Injectable } from '@angular/core';
 import { environment } from '@env/environment';
 
 @Injectable({
-  providedIn: 'root'
+  providedIn: 'root',
```
Comment (pbolduc) [inline src/frontend/citizen-portal/src/app/core/services/logger.service.ts:6]: Where is there a trailing comma?

File: src/frontend/citizen-portal/src/app/core/services/logger.service.ts (hunk 2 above)
Comment (pbolduc) [inline src/frontend/citizen-portal/src/app/core/services/logger.service.ts:9]: commented code

File: src/frontend/citizen-portal/src/app/core/services/toast.service.ts [hunk 4]
```diff
@@ -2,42 +2,54 @@ import { Injectable } from '@angular/core';
 import { MatSnackBar, MatSnackBarConfig } from '@angular/material/snack-bar';
 
 @Injectable({
-  providedIn: 'root'
+  providedIn: 'root',
```
Comment (pbolduc) [inline src/frontend/citizen-portal/src/app/core/services/toast.service.ts:5]: Comma?

File: src/frontend/citizen-portal/src/app/core/services/window-ref.service.ts [hunk 5]
```diff
@@ -5,10 +5,10 @@ function getWindow(): any {
 }
 
 @Injectable({
-  providedIn: 'root'
+  providedIn: 'root',
```
Comment (pbolduc) [inline src/frontend/citizen-portal/src/app/core/services/window-ref.service.ts:8]: comma

File: src/frontend/citizen-portal/src/app/core/services/window-ref.service.ts [hunk 6]
```diff
@@ -5,10 +5,10 @@ function getWindow(): any {
...
 })
 export class WindowRefService {
-  constructor() { }
+  // constructor() { }
```
Comment (pbolduc) [inline src/frontend/citizen-portal/src/app/core/services/window-ref.service.ts:11]: commented code

File: src/frontend/citizen-portal/src/app/modules/dashboard/components/dashboard-header/dashboard-header.component.ts [hunk 7]
```diff
@@ -1,12 +1,18 @@
...
+  EventEmitter,
+  Input,
+} from '@angular/core';
 
 @Component({
   selector: 'app-dashboard-header',
   templateUrl: './dashboard-header.component.html',
   styleUrls: ['./dashboard-header.component.scss'],
-  changeDetection: ChangeDetectionStrategy.OnPush
+  changeDetection: ChangeDetectionStrategy.OnPush,
```
Comment (pbolduc) [inline src/frontend/citizen-portal/src/app/modules/dashboard/components/dashboard-header/dashboard-header.component.ts:13]: comma

File: src/frontend/citizen-portal/src/app/modules/dashboard/components/dashboard/dashboard.component.ts [hunk 8]
```diff
@@ -70,7 +70,9 @@ export class DashboardComponent implements OnInit {
     }
   }
 
-  public onLogout() {}
+  public onLogout() {
+    console.log('onLogout');
```
Comment (pbolduc) [inline src/frontend/citizen-portal/src/app/modules/dashboard/components/dashboard/dashboard.component.ts:74]: Why are we using console here.. angular has a logging service.

**Review by pbolduc — COMMENTED** [review]
I stopped adding comments to this PR. There are a number of constructor methods that are commented out, extra trailing commas, use of console.log directly instead of using angular service.

**Review by CarolCarpenterJustice — APPROVED** [review]
Will update code later

**Pull request merged**